test/subgraph-tests/orders.test.ts

```diff
@@ -7,7 +7,7 @@ describe('Order fetching logic from subgraph', () => {
     console.log('Test running successfully');
     const orderId = '0xb160ae39e7a45b21fb8f247fb11f551f996ed90d3eb9a6263e49b98827e1fc4b';
 
-    const order = await getOrderById(chain, orderId);
+    const order = await getOrderById(chain, orderId, undefined);
```

Comment:
why are you sending a extra params? how is it working this [function](https://github.com/Parifi/parifi-sdk/blob/d624a9519b7846a1e690ef6aca3f0bb2efc9bb97/src/subgraph/orders/index.ts#L54) only recieve 2 params

Reply:
done